run_all.sh

```diff
@@ -1,16 +1,69 @@
 ###
-### Our Data Pipeline - This shell script contains all of the Python scripts that we use to go from
+### Our Data Pipeline - This shell script runs all of the Python scripts that we use to go from
 ### the source Chicago Data Portal CSV export to the final CSV and JSON files we use, including
 ### overall rankings, rankings by property type, and separate JSON files of the city-wide and
 ### property type specific averages, medians, and so forth
 ###
 
 # /bin/bash
 
-python3 -m src.data.scripts.clean_and_pare_down_data_all_years
+# Color codes for output
```

Comment:
Adding some color helps make clear what's going on, making this script more of a proper data pipeline than just a script that runs three files.

Reply:
nice addition!